### ISSUE TITLE
[Expo SQLite] Getting warning : `new NativeEventEmitter()` was called with a non-null argument without the required `addListener` 

### ISSUE THREAD

**Issue by LagoonProject** [expo/expo#25354]
### Minimal reproducible example

https://github.com/LagoonProject/ExpoSLiteWarningDebug

### Summary

When creating a table with Expo SQLite in a react-native app (not Expo app) I get the following warning on Android only (not ios): 

```
 WARN  `new NativeEventEmitter()` was called with a non-null argument without the required `addListener` method.
 WARN  `new NativeEventEmitter()` was called with a non-null argument without the required `removeListeners` method.

```

Using the following dependencies : 

```
  "dependencies": {
    "expo": "^49.0.0",
    "expo-sqlite": "^11.6.0",
    "react": "18.2.0",
    "react-native": "0.72.6"
  },
```
Thanks.

### Environment

  expo-env-info 1.0.5 environment info:
    System:
      OS: macOS 13.5.1
      Shell: 5.9 - /bin/zsh
    Binaries:
      Node: 20.5.1 - /opt/homebrew/bin/node
      Yarn: 1.22.19 - /opt/homebrew/bin/yarn
      npm: 9.8.0 - /opt/homebrew/bin/npm
    Managers:
      CocoaPods: 1.12.1 - /usr/local/bin/pod
    SDKs:
      iOS SDK:
        Platforms: DriverKit 22.4, iOS 16.4, macOS 13.3, tvOS 16.4, watchOS 9.4
      Android SDK:
        API Levels: 29, 33
        Build Tools: 30.0.3, 33.0.0
        System Images: android-29 | Google APIs ARM 64 v8a, android-33 | Google APIs ARM 64 v8a, android-33 | Google Play ARM 64 v8a
    IDEs:
      Android Studio: 2022.3 AI-223.8836.35.2231.10671973
      Xcode: 14.3.1/14E300c - /usr/bin/xcodebuild
    npmPackages:
      expo: ^49.0.0 => 49.0.18 
      react: 18.2.0 => 18.2.0 
      react-native: 0.72.6 => 0.72.6 
    Expo Workflow: bare

**Comment by Kudo** [comment]
hi there! thanks for reporting the issue. i'm sorry that we introduced this regression when introducing new features. the issue is actually resolved by newer versions (still in alpha). given the warning doesn't have an impact toward release builds. does that work for you if to downgrade the version to `expo-sqlite@11.3.3`?

**Comment by LagoonProject** [comment]
> hi there! thanks for reporting the issue. i'm sorry that we introduced this regression when introducing new features. the issue is actually resolved by newer versions (still in alpha). given the warning doesn't have an impact toward release builds. does that work for you if to downgrade the version to `expo-sqlite@11.3.3`?

Yes, downgrading to v11.3.3 indeed solves the warning, thanks for your answer!
